lib/dm-rails/setup.rb

```diff
@@ -5,25 +5,33 @@
 
 module Rails
   module DataMapper
+    DATABASE_URL = ENV['DATABASE_URL']
```

Comment:
@mbj I guess this would allow us to set `DATABASE_URL` within our env.rb that we use for (12 factor)[http://12factor.net/] apps we design.